include/open62541/server.h

```diff
@@ -1553,6 +1553,71 @@ UA_Server_updateCertificate(UA_Server *server,
                             UA_Boolean closeSessions,
                             UA_Boolean closeSecureChannels);
 
+#ifdef UA_GENERATED_NAMESPACE_ZERO_FULL
```

Comment:
Don't add this per default to namespace full.
It should be a separate configuration option.

Reply:
Fixed it.